pkgs/by-name/ma/mattermost/package.nix

```diff
@@ -32,14 +34,54 @@ buildGoModule rec {
     '';
   });
 
-  webapp = fetchurl {
-    url = "https://releases.mattermost.com/${version}/mattermost-${version}-linux-amd64.tar.gz";
-    hash = "sha256-yG5GDeuCHv95e+b2xi/UYiCGkV+I3aqj13Qh/YbyOWQ=";
-  };
+  webapp = buildNpmPackage rec {
+    pname = "mattermost-webapp";
+    inherit version src;
+
+    sourceRoot = "${src.name}/webapp";
+
+    # Fix build dependency conflicts
+    patchedPackageJSON = runCommand "package.json" { } ''
+      ${jq}/bin/jq '.devDependencies.ajv = "8.17.1" |
```

Comment:
Is it possible to use `lib.getExe jq` instead?

Reply:
Or pass as buildInputs to that runCommand arg and use jq directly.